timesketch/frontend/src/components/Sigma/SigmaRuleDetail.vue

```diff
@@ -0,0 +1,66 @@
+<!--
+Copyright 2021 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <strong>{{ rule.id }}</strong>
+    <ul class="content-list">
+      <li v-for="(value, name) in rule" v-bind:key="name" style="padding:10px;border-bottom:none;cursor:pointer;">
+        <div>
+          <b>{{ name }}</b
+          >: <code>{{ value }}</code>
```

Comment:
Can we make this a table instead? That will fit into the UI a bit better seeing that the event details is also a table.